scripts/grest-helper-scripts/setup-grest.sh

```diff
@@ -121,7 +121,7 @@ SGVERSION=1.0.8 # Using versions from 1.0.5-1.0.9 for minor commit alignment bef
       sed -e "s@NWMAGIC=.*@NWMAGIC=${NWMAGIC}@" -i "${CRON_SCRIPTS_DIR}/${job}.sh" &&
       sed -e "s@EPOCH_LENGTH=.*@EPOCH_LENGTH=${EPOCH_LENGTH}@" -i "${CRON_SCRIPTS_DIR}/${job}.sh" &&
       sed -e "s@PROM_URL=.*@PROM_URL=http://${PROM_HOST}:${PROM_PORT}/metrics@" -i "${CRON_SCRIPTS_DIR}/${job}.sh"
-      sed -e "s@CCLI_PATH=.*@CCLI_PATH=$(which cardano-cli)@" -i "${CRON_SCRIPTS_DIR}/${job}.sh"
+      sed -e "s@CCLI_PATH=.*@CCLI_PATH=${CCLI}}@" -i "${CRON_SCRIPTS_DIR}/${job}.sh"
       sed -e "s@CARDANO_NODE_SOCKET_PATH=.*@CARDANO_NODE_SOCKET_PATH=${CARDANO_NODE_SOCKET_PATH}@" -i "${CRON_SCRIPTS_DIR}/${job}.sh"
```

Comment:
Should replace this with ${SOCKET} as well

Reply:
actually, never mind, CARDANO_NODE_SOCKET_PATH is fine to use instead of SOCKET 👍